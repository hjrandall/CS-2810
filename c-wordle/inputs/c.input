sauc[e]
b[r][i]n[e]
