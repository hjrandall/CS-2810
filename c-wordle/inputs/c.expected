266: gride
265: grize
263: gripe
263: tride
260: grike
260: grime
260: tripe
258: pride
257: prize
257: trike
255: trite
252: prime
245: frize
244: write
238: drive
